tsconfig.build-es6.json

```diff
@@ -3,7 +3,7 @@
   "compilerOptions": {
     "target": "es6",
     "module": "es6",
-    "moduleResolution": "classic",
+    "moduleResolution": "node",
```

Comment:
Why was this changed?

Reply:
Because the build wasn't working without it.

Reply:
But this will not work for browsers.

Reply:
I thought that the `target` and `module` parts are the important ones. Feel free to fix it.

Reply:
Actually, you are right. There is this written in the docs:

```
If not specified, the default is [Node](https://www.typescriptlang.org/docs/handbook/module-resolution.html#node) for --module commonjs, and [Classic](https://www.typescriptlang.org/docs/handbook/module-resolution.html#classic) otherwise (including when [module](https://www.typescriptlang.org/tsconfig#module) is set to amd, system, umd, es2015, esnext, etc.).
```

so I assumed that it should be `classic`, but yeah - it shoul be `node`. I guess this turned out to be an issue after we've added a dependency which couldn't be imported.